test/runtests.jl

```diff
@@ -62,6 +62,14 @@ Distributed.addprocs(5)
                 @test isfile(joinpath(path, "tag=a", "date=2017-01-01", "TestPkg", "bar.jlso"))
             end
 
+            @testset "NamedTuple tags" begin
+                tags = (tag1="some", tag2="thing")
+                with_checkpoint_tags(tags) do
```

Comment:
in the downstream example the `tags` were splatted like `tags...`

Edit: I just went back to look and noticed you changed it to now slurp them

Reply:
Yeah annoyingly if you splat a named tuple you only get the values